tests/foreman/api/test_product.py

```diff
@@ -188,11 +153,11 @@ def test_positive_update_description(description, module_org, module_target_sat)
 
 @pytest.mark.tier1
 def test_positive_update_name_to_original(module_org, module_target_sat):
```

Comment:
The original `test_positive_update_name` was parametrized to update name with different char-sets. Do we want to keep that parametrization?
```suggestion
@pytest.mark.parametrize('new_name', **parametrized(valid_data_list()))
def test_positive_update_name_to_original(new_name, module_org, module_target_sat):
```

Reply:
Yes we can use valid data set, I will try locally and update code.